docs/tutorials/optimizers_conditionalgradient.ipynb

```diff
@@ -0,0 +1,558 @@
+{
```

Comment:
<p>Could you re-use the same model and just run <a href="https://www.tensorflow.org/api_docs/python/tf/keras/backend/clear_session" target="_blank">https://www.tensorflow.org/api_docs/python/tf/keras/backend/clear_session</a> at the beginning of this section</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/addons/pull/590/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>

Reply:
<p>I have faced some issue with using "tf.keras.backend.clear_session()" from this link. It seems that the variables in the model has not been cleared out and the model keeps on optimizing based on previous updated variable value, which makes the model second time with using SGD start from pretty high accuracy value. Not sure for the reason. Sorry about this. </p>

Reply:
Hmmm not sure of the reason for that. But could you revert back to `model_1` and `model_2` then.